components/authz-service/testhelpers/testhelpers.go

```diff
@@ -358,3 +397,45 @@ func WaitForWithTimeout(t *testing.T, f func() bool, timeout time.Duration, mess
 		time.Sleep(time.Millisecond * 10)
 	}
 }
+
+type mockProjectUpdateManager struct{}
+
+func NewMockProjectUpdateManager() *mockProjectUpdateManager {
+	return &mockProjectUpdateManager{}
+}
+
+func (*mockProjectUpdateManager) Cancel() error {
+	return nil
+}
+
+func (*mockProjectUpdateManager) Start() error {
+	return nil
+}
+
+func (*mockProjectUpdateManager) ProcessFailEvent(eventMessage *automate_event.EventMsg) error {
+	return nil
+}
+
+func (*mockProjectUpdateManager) ProcessStatusEvent(eventMessage *automate_event.EventMsg) error {
+	return nil
+}
+
+func (*mockProjectUpdateManager) Failed() bool {
+	return false // manager.stage.Failed
+}
+
+func (*mockProjectUpdateManager) FailureMessage() string {
+	return "" // manager.stage.FailureMessage
+}
+
+func (*mockProjectUpdateManager) PercentageComplete() float64 {
+	return 0
+}
+
+func (*mockProjectUpdateManager) EstimatedTimeComplete() time.Time {
+	return time.Time{}
+}
+
+func (*mockProjectUpdateManager) State() string {
+	return "not_running" // manager.stage.State
```

Comment:
By always passing "not_running" that removes the ingest service from the picture on this set of integration tests.